### PR TITLE
Rd 3288 eks provisioning

### PR DESCRIPTION
**Pull request by glukhman**

### CODE REVIEW

File: marketplace-helm/templates/deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      securityContext:
+        {{- toYaml $.Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ $svc }}-container
+          securityContext:
+            {{- toYaml $.Values.securityContext | nindent 12 }}
+          image: {{ $v.image }}
+          imagePullPolicy: {{ $.Values.image.pullPolicy }}
+          ports:
+            - name: http
```
Comment (warolv) [inline marketplace-helm/templates/deployments.yaml:38]: export this section also to values.yaml

**Review by warolv — CHANGES_REQUESTED** [review]

**Review by warolv — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -54,6 +54,183 @@ docker run -d --name cloudify-marketplace-plugins \
...
+
+1. Copy the following content to a `values.yaml` file. Get the `host`, 
+   `master_username` and `master_password` values from the capabilities of 
+   your RDS deployment.
+   ```YAML
+   ingress:
+     enabled: true
+   services:
+     plugins:
+       image: glukhman/cloudify-marketplace:plugins
```
Comment (mateumann) [inline README.md:214]: I'm not sure if tagging is the best way do differentiate images for our 3 services, I would rather use three separate names (e.g. `cloudify-marketplace-plugins`, `cloudify-marketplace-blueprints` etc.)

File: marketplace-helm/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  {{- end }}
+{{- end }}
+
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
```
Comment (mateumann) [inline marketplace-helm/templates/ingress.yaml:17]: Do we expect this chart to be used in versions of k8s < 1.19 (released 2020-08-26)?

File: README.md (hunk 2 above)
Comment (glukhman) [reply README.md:214]: That's going to change as soon as the images are in their permanent place. I'll oren a separate PR for it

File: marketplace-helm/templates/deployments.yaml (hunk 1 above)
Comment (glukhman) [reply marketplace-helm/templates/deployments.yaml]: done

File: marketplace-helm/templates/ingress.yaml (hunk 3 above)
Comment (glukhman) [reply marketplace-helm/templates/ingress.yaml:17]: As of now, the default ingress API in our k8s plugin is v1beta1, which works with k8s 1.18.
As soon as we change it, the chart can indeed become simpler

**Pull request merged**